src/darwin/Framework/CHIP/MTRDeviceController_XPC.mm

```diff
@@ -41,11 +43,13 @@ @interface MTRDeviceController_XPC ()
 @implementation MTRDeviceController_XPC
 
 @synthesize uniqueIdentifier = _uniqueIdentifier;
+@synthesize nodeIDToDeviceMap = _nodeIDToDeviceMap;
+@synthesize deviceMapLock = _deviceMapLock;
```

Comment:
Why do you need these? They are already in the base implementation

Reply:
Yup, not needed, they'll create shadows.